sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -29,7 +29,10 @@ page at http://checkstyle.sourceforge.net/config.html -->
     <!-- Checks that there are no tab characters in the file. -->
   </module>
 
-  <module name="NewlineAtEndOfFile"/>
+  <module name="NewlineAtEndOfFile">
+    <!-- windows can use \n\r vs \n, so enforce the most used one ie UNIx style -->
+    <property name="lineSeparator" value="lf" />
+  </module>
```

Comment:
I'm not sure I fully get the question but tools module was failing cause of that on windows so just fixed this validation to take unix style EOL.